networking/src/Models/ITeamAnswerContent.ts

```diff
@@ -12,14 +12,15 @@ export interface IExtractedAnswer {
   type: AnswerType;
 }
 export interface INormAnswer {
-  value: string | number;
-  type: AnswerType;
```

Comment:
new `INormAnswer` per the Pr description

Reply:
hmm, can we have a mix between answer types, e.i. both number and expression types have values?

Reply:
Also, one other way to look at this is having a based class and then subclasses for each type of answers. Then, base class can define equal method which each child class needs to override it. Let me know if you need any help or prefer an example and happy to write up something in playground. We should also restrict the types, like array of numbers as answer should be a separate answer type vs. just one answer. For example, what kind of questions have an array of expressions as answer? I can see it for string and numbers, but not expressions. It also should be a separate type which then gives us better knowledge in UI how to handle it. Like we can update the text editor in answers that after each space, shows the number as the whole with background.

Reply:
Actually to help better understand the proposal above, here's a code snippet for it:

```typescript
enum AnswerType {
    Number,
    Numbers,
    String,
    Strings,
    Expression,
}

abstract class BaseAnswer<T> {
    answerType: AnswerType
    value: T | null = null

    constructor(answerType: AnswerType, value: T | null = null) {
        this.answerType = answerType
        this.value = value
    }

    abstract isEqualTo(answer: BaseAnswer<T>): Boolean
}

class NumberAnswer extends BaseAnswer<number> {
    isEqualTo(answer: NumberAnswer): Boolean {
        return this.value == answer.value
    }
}

let a = new NumberAnswer(AnswerType.Number, 2)
let b = new NumberAnswer(AnswerType.Number, 2)
```